decidim-debates/config/locales/en.yml

```diff
@@ -220,5 +221,39 @@ en:
         description: Number of debates created
         object: debates
         title: Debates
+    open_data:
+      help:
+        debate_comments:
+          alignment: If this comment was a favour, against or neutral
+          author: The name of the participant that made this comment
+          body: The comment itself
+          commentable_id: The unique id of the commentable
+          commentable_type: The type of the commentable (if it was a result, a proposal, etc.)
+          created_at: The date when this comment was created
+          depth: The place where this comment is in the three of comments (if it is an answer or an answer of an answer)
+          id: The id for this comment
+          locale: The locale (language) that the participant had when leaving this comment
+          root_commentable_url: The URL of the resource that ties to this comment
+          user_group: The name of the user group that made this comment (if any)
+        debates:
```

Comment:
I think we need to add help instructions for conclusions and closed_at fields. What do u think?

Reply:
Done at 3abd24e8b6